lib/multiple_view_date_range_picker.dart

```diff
@@ -433,44 +462,26 @@ class _MultipleViewDateRangePickerState extends State<MultipleViewDateRangePicke
           ],
         ),
       );
-    } else {
+    } else if (!bottomViewTabletConfirmationButtonsOverflow) {
       return Padding(
-        padding: const EdgeInsets.all(16),
+        padding: EdgeInsets.all(tabletDateRangePickerBottomPadding),
```

Comment:
Please use `const` for widget if possible

Reply:
variables inside `State` object cannot be `const`, only `final`. Hence, `const` cannot be applied here.

Reply:
So let's give it a `const` value. If you don't do this, the widget tree may be re-rendered when the state changes.

Reply:
Done